README.md

```diff
@@ -58,6 +58,30 @@ Install Nagios and setup the password for your nagiosadmin user.
       - user: nagiosadmin
         pass: S3cre7Passw0rd
 
+## Upgrading
+
+The role has automatic upgrade when you change the version with var: 
+
+    nagios_version: 4.3.2
+
+But the upgrade requires to remove some old files to, check [build-nagios.yml](tasks/build-nagios.yml) for details.
+
+## Supported versions
```

Comment:
I have removed the lines, and created #9 to not forgot about 16.04. 

The test was required when I added automatic tests for https://github.com/CoffeeITWorks/ansible_nagios4_server_plugins, because this role uses ansible-role-nagios to have an installed nagios version. 

Kind regards.
Pablo.


Thanks for your patience!